instrumentation/all/opentelemetry-instrumentation-all.gemspec

```diff
@@ -26,18 +26,18 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.5.0'
 
-  spec.add_dependency 'opentelemetry-instrumentation-concurrent_ruby', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-dalli', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-ethon', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-excon', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-faraday', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-mysql2', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-net_http', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-rack', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-redis', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-restclient', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-sidekiq', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-sinatra', '~> 0.8.0'
+  spec.add_dependency 'opentelemetry-instrumentation-concurrent_ruby', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-dalli', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-ethon', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-excon', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-faraday', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-mysql2', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-net_http', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-rack', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-redis', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-restclient', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-sidekiq', '~> 0.9.0'
+  spec.add_dependency 'opentelemetry-instrumentation-sinatra', '~> 0.9.0'
 
```

Comment:
🤔 I think we need to add some new gems here?

Reply:
Ahh, yeah.  Good catch.

Reply:
Added